bindings/python/pydeck/docs/README.md

```diff
@@ -1,4 +1,19 @@
 pydeck API Documentation
 =========
 
-TODO This will be a table of contents for the pydeck documentation
+## Table of contents
+#### [Deck](https://github.com/uber/deck.gl/blob/master/bindings/python/pydeck/docs/deck.md)
```

Comment:
Use url from the root of this repo `/bindings/...` otherwise this is will link to the wrong file on a release branch.